src/util/types.h

```diff
@@ -126,6 +126,9 @@ assetToString(const Asset& asset)
     case stellar::ASSET_TYPE_CREDIT_ALPHANUM12:
         assetCodeToStr(asset.alphaNum12().assetCode, r);
         break;
+    case stellar::ASSET_TYPE_POOL_SHARE:
```

Comment:
This made me realize that you need to add new overloads to XDRCereal.h for `TrustLineAsset` and `ChangeTrustAsset`. I would recommend unifying the logging of all asset types in a single template overload.

Reply:
Done. I tested this change with `xdr_to_string` and looks like it works.